src/components/ButtonGroupPicker/styled/buttonGroup.js

```diff
@@ -0,0 +1,84 @@
+/* stylelint-disable no-descending-specificity */
+import styled from 'styled-components';
+
+const StyledButtonGroup = styled.div`
+    display: inline-flex;
+
+    span {
```

Comment:
We should not do introspection on the component's styles.  Instead, to use nested selectors we should make the `span` a styled